phpstan.neon

```diff
@@ -51,15 +52,19 @@ parameters:
     - '#Parameter \#1 \$data of function bin2hex expects string, string\|false given\.#'
     - '#Parameter \#1 \$query of method Illuminate\\Database\\Eloquent\\Builder<App\\Models\\Attendance>::selectSub\(\) expects Closure\|Illuminate\\Database\\Eloquent\\Builder<App\\Models\\Attendance>\|string, Illuminate\\Database\\Eloquent\\Builder<App\\Models\\User> given\.#'
     - '#Parameter \#1 \$str of function strtolower expects string, string\|null given\.#'
+    - '#Parameter \#1 \$string of function strlen expects string, string\|false given\.#'
+    - '#Parameter \#1 \$user of method Illuminate\\Foundation\\Testing\\TestCase::actingAs\(\) expects Illuminate\\Contracts\\Auth\\Authenticatable, mixed given\.#'
```

Comment:
Should we just ignore the `tests` directory entirely?

Reply:
If it keeps requiring ignores then sure.